### PR TITLE
Add bootstrap

### PR DESCRIPTION
**Pull request by intellectronica**

### CODE REVIEW

**Comment by cyberstar** [comment]
LGTM

File: base.html [hunk 1]
```diff
@@ -27,6 +27,8 @@
             }
         </style>
         {% render_block "css" %}
+        <link rel="stylesheet" href="{% static 'css/bootstrap.min.css' %}" />
```
Comment (FinalAngel) [inline base.html:30]: you might want to place bootstrap before the render_block "css"

**Comment by FinalAngel** [comment]
LGTM except for the style placement (which probably won't cause problems atm)

File: base.html (hunk 1 above)
Comment (intellectronica) [inline base.html:30]: thanks, i'll move it

**Pull request merged**